### PR TITLE
Track missing alpha channels in colors

### PR DESCRIPTION
**Pull request by nex3**
See sass/sass#2831

### CODE REVIEW

File: lib/src/value/color.dart [hunk 1]
```diff
@@ -89,6 +89,8 @@ class SassColor extends Value {
   /// indicates a [missing] component.
   ///
   /// [missing]: https://www.w3.org/TR/css-color-4/#missing
+  ///
+  /// @nodoc
```
Comment (Goodwine) [inline lib/src/value/color.dart:93]: I'm curious why the `@nodoc` was added

**Review by Goodwine — COMMENTED** [review]
I'm guessing no CHANGELOG was added because this is a feature branch, but should there be tests anywhere for these?

File: lib/src/value/color.dart (hunk 1 above)
Comment (nex3) [reply lib/src/value/color.dart:93]: All of the channel-specific getters are intended to be package-private, I just forgot to add `@nodoc` earlier.

**Comment by nex3** [comment]
> I'm guessing no CHANGELOG was added because this is a feature branch, but should there be tests anywhere for these?

We're working on tests separately in the feature branch of the sass-spec repo, but because the tests are fairly holistic we're getting the implementation mostly in-place first.

**Review by Goodwine — APPROVED** [review]

**Pull request merged**